README.md

```diff
@@ -64,3 +64,40 @@ Sync the overlay
 ```
 emerge --sync
 ```
+
+### Updating the Prefix
+#### Packages
+Updating packages can be as easy as
+```
+emerge --sync
+emerge
+```
+If you run into problems, usually a newer ebuild is not suited to build in a prefix environment.
+Try to mask latest versions:
+
+Create a mask file if not existing and mask newer versions from thin provisioning tools greater or equal to 0.7.6:
+```
+touch /path/to/prefix/etc/portage/package.mask
+echo ">=sys-block/thin-provisioning-tools-0.7.6" >> /path/to/prefix/etc/portage/package.mask
```

Comment:
Use `$EPREFIX`, so it's consistent with the above? Also, the `touch` is not needed?

```
export EPREFIX=/path/to/prefix
echo ">=sys-block/thin-provisioning-tools-0.7.6" >> $EPREFIX/etc/portage/package.mask

Reply:
agreed